src/js/plugins/tts/plugin.tts.js

```diff
@@ -108,13 +110,15 @@ BookReader.prototype.ttsStart = function () {
         this.switchMode(this.constMode1up);
 
     this.$('.BRicon.read').addClass('unread');
+    this.ttsSendAnalyticsEvent('Start', null, this.ttsEngine.opts.bookLanguage || 'null');
     this.ttsEngine.start(this.currentIndex(), this.getNumLeafs());
 };
 
 // ttsStop()
 //______________________________________________________________________________
 BookReader.prototype.ttsStop = function () {
     this.$('.BRicon.read').removeClass('unread');
+    this.ttsSendAnalyticsEvent('Stop', null, this.ttsEngine.opts.bookLanguage || 'null');
```

Comment:
Why do we track stop events? And does stop even mean stop playing or leave audioreader mode?

Reply:
Mostly because "why not"? We can remove it if we find it's not useful, but we can't add it in if we find out we do need it.